packages/phone-number-privacy/signer/.env

```diff
@@ -28,12 +28,20 @@ SERVICE_NAME='odis-signer'
 ODIS_SIGNER_SERVICE_URL=https://staging-pgpnp-signer0.azurefd.net
 ODIS_COMBINER_SERVICE_URL=https://us-central1-celo-phone-number-privacy-stg.cloudfunctions.net
 ODIS_BLOCKCHAIN_PROVIDER=https://alfajores-forno.celo-testnet.org
+# ODIS_BLOCKCHAIN_PROVIDER=https://staging-forno.celo-networks-dev.org/
+ODIS_PUBLIC_POLYNOMIAL_VAR_FOR_TESTS=STAGING_PHONE_NUMBER_PRIVACY_POLYNOMIAL
+# TODO(ODIS 2.0.0 e2e fix): this needs to be updated
+ODIS_PUBLIC_DOMAINS_POLYNOMIAL_VAR_FOR_TESTS=STAGING_PHONE_NUMBER_PRIVACY_POLYNOMIAL
+ODIS_PUBLIC_DOMAINS_PUBKEY_VAR_FOR_TESTS=STAGING_DOMAINS_PUBKEY
+# TODO(ODIS 2.0.0 e2e fix): this is wrong and is the same as the PNP staging pubkey for now
+STAGING_DOMAINS_PUBKEY=7FsWGsFnmVvRfMDpzz95Np76wf/1sPaK0Og9yiB+P8QbjiC8FV67NBans9hzZEkBaQMhiapzgMR6CkZIZPvgwQboAxl65JWRZecGe5V3XO4sdKeNemdAZ2TzQuWkuZoA
```

Comment:
@eelanagaraj we're using the same keys in staging for domains and pnp so this is actually correct I think